spec/spec.md

```diff
@@ -29,11 +29,15 @@ Participate:
 
 ## Abstract
 
-The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of Verifiable Credentials between Wallets and Verifiers/RPs.
+The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between digital [[ref: Wallets]] and credential [[ref: Verifiers]].
```

Comment:
It might be more clear to be a "set of requirements" than a set of specifications, since in the next sentence we say that we're not a specification :)

Reply:
Or even "a set of requirements against existing specifications" perhaps?

Reply:
+1

Reply:
ah great!! updating.